src/components/AppMixin/AppMixin.js

```diff
@@ -59,6 +59,14 @@ var AppMixin = {
             var start = timestampToYear(response.data[key].start_date);
             var end = timestampToYear(response.data[key].end_date);
 
+            // Stopgap measure to deal with the fact that experiment string formats
+            // vary between climdex files ("historical, rcp26") and GCM outputs
+            // ("historical,rcp26"). Formats experiment strings to include a space.
+            // This formatting is undone to run queries against the database by
+            // ce-backend.guessExperimentFormatFromVariable()
+            // TODO: remove this when no longer needed.
+            var normalizedExp = String(response.data[key].experiment).replace(',r', ', r');
```

Comment:
A more generic or flexible substitution here does not give us any more functionality, because later on we have to *exactly* undo this substitution in order to run queries against the database with whatever the original database string was, which is of course the one in the database. If there were multiple possible pre-standardization scenario strings (zero spaces, three spaces, underscores, whatever) for each standard one, we would not necessarily be able to recover the original string when we needed it for the back end.

If at some point we have more than two possible spellings of each emissions scenario description, we'll need a dictionary or something else more complex.

Reply:
Alternatively, and in my opinion better, the backend needs to be smarter about these strings, performing a similar normalization as part of the matching. This should not be hard. And it follows the [Robustness Principle](https://en.wikipedia.org/wiki/Robustness_principle), "Be conservative in what you send, be liberal in what you accept".

Reply:
We have a [data prep issue](https://github.com/pacificclimate/climate-explorer-data-prep/issues/48) about experiment strings, should I add a backend issue too / instead?

Reply:
Yes, please.

Reply:
Here it is: https://github.com/pacificclimate/climate-explorer-backend/issues/76